src/proto/ctl/smd.proto

```diff
@@ -76,10 +76,11 @@ message BioHealthResp {
 }
 
 enum NvmeDevState {
-	NORMAL = 0;	// Device is in a normal operational state
-	NEW = 1;	// Device is new and is not yet in-use
-	EVICTED = 2;	// Device is faulty and has been evicted
-	UNPLUGGED = 3;	// Device has been physically removed
+	UNKNOWN   = 0; // Device state is unknown, zer6o value
```

Comment:
nit - typo
```suggestion
	UNKNOWN   = 0; // Device state is unknown, zero value
```

Reply:
fix for this applied on https://github.com/daos-stack/daos/pull/12944